### PR TITLE
chore: embed-services package

### PR DESCRIPTION
**Pull request by josephaxisa**
This is just a skeleton of the package that will contain the service layer for looker embed components.

### CODE REVIEW

File: packages/embed-services/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "main": "lib/index.js",
+  "module": "lib/esm/index.js",
+  "sideEffects": false,
+  "typings": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "private": false,
+  "publishConfig": {
+    "access": "public"
```
Comment (jkaster) [inline packages/embed-services/package.json:14]: I think you want this private for now

File: packages/embed-services/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2022 Looker Data Sciences, Inc.
```
[the remark this answers is not in the document]
Comment (jkaster) [reply packages/embed-services/src/index.ts:5]: Yes, just let the linter do its thing. It's based on the `LICENSE` from the main folder, which should be updated to 2023, then your local `LICENSE` file matches.

**Review by jkaster — CHANGES_REQUESTED** [review]
You should update the main repo LICENSE file for the current copyright year

File: packages/embed-services/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  "version": "0.0.1-alpha",
+  "description": "Looker Embed Services",
+  "main": "lib/index.js",
+  "module": "lib/esm/index.js",
+  "sideEffects": false,
+  "typings": "lib/index.d.ts",
+  "files": [
+    "lib"
+  ],
+  "private": false,
```
Comment (jkaster) [inline packages/embed-services/package.json:12]: should this be `true` or `false`?

**Review by jkaster — COMMENTED** [review]
got a question before approval

File: packages/embed-services/package.json (hunk 3 above)
Comment (josephaxisa) [reply packages/embed-services/package.json]: good catch, thanks.

**Review by jkaster — APPROVED** [review]

**Pull request merged**